components/Banner/src/MDCBannerView.h

```diff
@@ -62,6 +77,6 @@ __attribute__((objc_subclassing_restricted)) @interface MDCBannerView : UIView
 
  The default value is light grey.
  */
-@property(nonatomic, nonnull, strong) UIColor *dividerColor;
+@property(nonatomic, readwrite, strong, nonnull) UIColor *dividerColor;
```

Comment:
Please move this change to a separate PR.

Reply:
Addressed in d457cf307816ca947981521e366f07aa2565dab8.

Reply:
Addressed in https://github.com/material-components/material-components-ios/pull/7575